src/schema/provenance_schema.yaml

```diff
@@ -270,7 +270,7 @@ ENTITIES:
     derivation:
       source: true
       target: true
-    properties:
+    properties: &shared_dataset_properties
       <<: *shared_properties
       <<: *shared_entity_properties
       <<: *doi_properties
```

Comment:
Since the `&shared_dataset_properties` already contains other shared properties, no need to specify them again:
```
<<: *shared_properties
<<: *shared_entity_properties
<<: *doi_properties
```

Reply:
Good catch, yes this would be redundant